gpcontrib/arenadata_toolkit/arenadata_toolkit--1.4--1.5.sql

```diff
@@ -0,0 +1,119 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.4--1.5.sql */
+
+/*
+ This is part of arenadata_toolkit API for ADB Bundle.
+ This function collects information in db_files_current and db_files_history tables.
+ */
+CREATE OR REPLACE FUNCTION arenadata_toolkit.adb_collect_table_stats()
+RETURNS VOID
+AS $$
+DECLARE
+	ops oid[] DEFAULT (SELECT array_agg(oid) FROM pg_opclass WHERE opcname = 'timestamp_ops');
+BEGIN
+	IF NOT EXISTS (
+		SELECT
+		FROM pg_partition pp
+		JOIN pg_partition_rule pr ON pr.paroid = pp.oid
+		WHERE
+			CASE
+				WHEN pp.parclass[0] = ANY (ops)

```

Comment:
Can we use `pp.parrelid = 'arenadata_toolkit.db_files_history'::regclass` instead of `pp.parclass[0] = ANY (ops)`?

Reply:
I think yes, and I guess we also can remove
```
			AND
			NOT pp.paristemplate
			AND
			pp.parkind = 'r'  --range
			AND
			pr.parrangestartincl = 't'
```
as far as we do not change the partitioning scheme of 'arenadata_toolkit.db_files_history'.